packages/edit-site/src/components/style-book/constants.ts

```diff
@@ -6,7 +6,46 @@ import { __ } from '@wordpress/i18n';
 /**
  * Internal dependencies
  */
-import type { StyleBookCategory } from './types';
+import type { StyleBookCategory, StyleBookColorGroup } from './types';
+
+export const STYLE_BOOK_COLOR_GROUPS: StyleBookColorGroup[] = [
+	{
+		slug: 'theme-colors',
+		title: __( 'Theme Colors' ),
+		origin: 'theme',
+		type: 'colors',
+	},
```

Comment:
I'm just mucking around, testing things out. 

It's working really well. The real-time updates of custom colors is 💯 

I think just theme gradients are missing?

```suggestion
	},
	{
		slug: 'theme-gradients',
		title: __( 'Theme Gradients' ),
		origin: 'theme',
		type: 'gradients',
	},
```

Reply:
I'd agree they are missing. I was just blindly following the design too literally until I got things rendering appropriately.

<img width="400" alt="Screenshot 2024-10-11 at 4 11 16 pm" src="https://github.com/user-attachments/assets/c8a4af92-2c68-42fc-ae85-13dd430e4ce0">

Happy to add them. It's easy to yank them if there was a reason they weren't in the design.

Reply:
> Happy to add them. It's easy to yank them if there was a reason they weren't in the design.

Oh, sorry I missed that. I assume that would have been an oversight not adding a theme's gradients in there 🤔

Anyway, so far it's looking swell.